scripts/qa_crowbarsetup.sh

```diff
@@ -1267,6 +1267,12 @@ EOF
     # setup_base_images.rb is for SUSE Cloud 1.0 and update_nodes.rb is for 2.0
     sed -i -e 's/\(rootpw_hash.*\)""/\1"$2y$10$u5mQA7\/8YjHdutDPEMPtBeh\/w8Bq0wEGbxleUT4dO48dxgwyPD8D."/' /opt/dell/chef/cookbooks/provisioner/recipes/setup_base_images.rb /opt/dell/chef/cookbooks/provisioner/recipes/update_nodes.rb
 
+    if [[ $hacloud = 1 ]] ; then
```

Comment:
Shouldn't we have extra option for this setup? I mean, is there other way to setup shared glance storage than NFS?